includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1080,6 +1080,26 @@ public function get_platform_checkout_eligibility() {
 		);
 	}
 
+	/**
+	 * Update platform checkout data
+	 *
+	 * @param array $data Data to update.
+	 *
+	 * @return array An array describing request result.
+	 *
+	 * @throws API_Exception - Error contacting the API.
+	 */
+	public function update_platform_checkout( $data ) {
```

Comment:
So, according to [this](690-gh-Automattic/woopay/files#diff-ac39b9f33c738e105ed936ad894e08300688e9db00a1d8a3c11f5a4426ea6e18) the merchant can update the webhook secret and the business profile in the same request, right?